components/brave_ads/browser/ads_p2a.h

```diff
@@ -7,19 +7,89 @@
 #define BRAVE_COMPONENTS_BRAVE_ADS_BROWSER_ADS_P2A_H_
 
 #include <string>
+#include <vector>
 
 class PrefService;
 class PrefRegistrySimple;
 
 namespace brave_ads {
 
+const std::vector<std::string> kP2AQuestionsList = {
```

Comment:
nit: `kP2aQuestionList` instead of `kP2aQuestionsList` as list implies there are more than 1